src/engine/driver/Driver.java

```diff
@@ -22,37 +24,18 @@ public class Driver {
 	 * @param args
 	 */
 	public static void main(String[] args) {
-		char delim = args[2].charAt(0);
-		String filePath = args[1];
-		String content = "";
-        try {
-            content = new String(Files.readAllBytes(Paths.get(filePath)));
-            System.out.println(content);
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-        
-        List<String> inputIntegersAsString = Arrays.asList(content.split(","));
-        
-        List<Integer> inputIntegers = new ArrayList<>();
-        
-        for(String s : inputIntegersAsString) {
-        	try {
-        	inputIntegers.add(Integer.valueOf(s));
-        	} catch (Throwable t) {
-        		t.printStackTrace();
-        	}
-        }
-        
-        UserDataInput data = new UserDataInput(delim, inputIntegers);
-        
-        User theUser = new User();
-        
-		ProtoUserDataStream outputData = theUser.sendUserRequest(data);
+		char delim = args[3].charAt(0);
+		String filePathIn = args[1];
```

Comment:
Why do we need filepath in?

Reply:
we need to know what our input and output file paths are.